app/assets/stylesheets/application.css.scss.erb

```diff
@@ -43,186 +43,224 @@ td, th {
-    }
+    width: 50%;
+    margin: 0px auto;
+    color: #666666;
+    font-size: 17px;
+  }
+  a {
+    margin: 30px 0px;
+  }
 
 }
 
-
 .grey-text {
-    color: #666666;
-        }
-.event_border{
+  color: #666666;
+}
+
+.event_border {
   border: 1px solid green !important;
   margin: 3% 5% !important;
   width: 90% !important;
   padding: 20px !important;
-    border-radius: 5px;
+  border-radius: 5px;
 }
-.create_input{
-    width: 50%;
+
+.create_input {
+  width: 50%;
 }
-.indicator{
+
+.indicator {
   display: none !important;
 }
-.create_form{
-    margin-top: 50px;
+
+.create_form {
+  margin-top: 50px;
 }
-.temps{
-    margin-top: 40px;
+
+.temps {
+  margin-top: 40px;
 }
-.temp_image{
-    margin-bottom: 20px !important;
 
+.temp_image {
+  margin-bottom: 20px !important;
 }
-.create_form input.select-dropdown{
-    margin:0 0 15px 0 !important;
+
+.create_form input.select-dropdown {
+  margin:0 0 15px 0 !important;
 }
 
-.domain{
-    margin-top:10px;
+.domain {
+  margin-top:10px;
 }
-.create{
-    background-color: #6B6A6A !important;
+
+.create {
+  background-color: #6B6A6A !important;
 }
 
-.fixer{
-    position:fixed !important;
-    background-color: rgba(0, 0, 0, 0.8) !important;
-    z-index: 1000;
+.fixer {
+  position:fixed !important;
+  background-color: rgba(0, 0, 0, 0.8) !important;
+  z-index: 1000;
 }
-.showpage-header{
-    height:20% !important;
+
+.showpage-header {
+  height:20% !important;
 }
-.our-sidenav{
-    padding: 0 !important;
-    margin: 0 !important;
+
+.our-sidenav {
+  padding: 0 !important;
+  margin: 0 !important;
 }
-.our-collection-item{
-    font-size: 14px;
+
+.our-collection-item {
+  font-size: 14px;
 }
-.our-search{
-    margin: 0 !important;
-    background-color: rgba(0, 0, 0, 0.64);
+
+.our-search {
+  margin: 1rem !important;
+  background-color: rgba(0, 0, 0, 0.5);
+  position: relative;
 }
-.before-scroll{
-    display:none;
-    padding-top: 5px;
+
+.search-caption{
+  background-color: rgb(247, 183, 51);
+  width: 200px;
+  max-width: 100%;
+  padding: .5rem;
+  font-weight: 500;
+}
+
+.before-scroll {
+  display:none;
+  padding-top: 5px;
 }
 
 .active.our-tab-color {
   background-color: #313131 !important;
   color: #ffffff !important;
 }
+
 .active.our-tab2-color {
   background-color: #313131 !important;
   color: #ffffff !important;
 }
+
 .create_color {
-    background-color: #F7F7F7;
+  background-color: #F7F7F7;
+  width: 100%;
+  height: 100%;
+  nav {
+    background-color: #FFFFFF !important;
+  }
+}
+
+.create_header_background {
+  background: url('<%= asset_path('best3.jpg') %>') center;
+  background-size: cover;
+  height: 350px;
+  position: relative;
+  overflow: hidden;
+  .creat-overlay {
     width: 100%;
     height: 100%;
-    nav{
-        background-color: #FFFFFF !important;
-    }
-}
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0.35);
+  }
+
+  h2 {
+    top: 100px;
+    position: relative;
+    color: #F7F7F7;
```

Comment:
@andela-megwali At some point in time, we will have to put all this hex value in a `variable.scss` file. Not for now though.